fineract-core/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java

```diff
@@ -102,6 +103,11 @@ public CommandProcessingResult executeCommand(final CommandWrapper wrapper, fina
         } else {
             idempotencyKey = idempotencyKeyResolver.resolve(wrapper);
         }
+        // Idempotency key lebgth validation
+        if (!StringUtils.isEmpty(idempotencyKey) && idempotencyKey.length() > 50) {
```

Comment:
This seems not right. If the idempotency key is too long, we should have a domain rule exception (HTTP 403) that describes the issue. The issue was when it happened the error message title and message was incorrect.

Reply:
Done! Exception updated to be as Domain rule exception

Reply:
You misunderstood me. I dont want you to check the length of the idempotency field. It will automatically throw a database error which will be translated to a domain rule exception. I would like you to debug out why wrong exception title and message was generated and fix it